USAGE.md

```diff
@@ -0,0 +1,142 @@
+Usage
+=====
+
+## Requirements
+
+You need at least Node `v4.0.0` and npm `2`.
+
+## Create your project
+Install the required tools: yo, gulp, bower
```

Comment:
I would add a small sentence like "You only need the tool corresponding with the technologies you'll choose with the generator" (don't want to scare a Babel guy with an install of typings)